deploy.py

```diff
@@ -664,6 +686,41 @@ def run(self):
         return 1
       return 0
 
+    if self.args.programmer == "bossac":
+      props = SUPPORTED_BOARDS[self.args.board]
+      # Call the Makefile in order to build and flash Tock OS using bossac
+      if self.args.tockos:
+        # The Arduino Nano requires a double tap on the reset button
+        # in order to enter bootloader mode
+        # We must wait for the user to confirm that
+        # the board is ready to load something on it
+        info("""Double tap the reset button on your Arduino Nano board
+            in order to begin the flashing process""")
+        info("Press [ENTER] when ready.")
+        _ = input()
+        self.checked_command(["make", "program"], cwd=props.path)
+
+      # Install the app using bossac
+      if self.args.application:
+        info("""Double tap the reset button on your Arduino Nano board
+            in order to load the application onto the board""")
+        info("Press [ENTER] when ready.")
+        _ = input()
+
+        self.checked_command(
+            [
+                "bossac",
```

Comment:
Please use long options when available as they are easier to understand/review and (normally) more stable overtime than the short ones.

Reply:
Yeah, I should've thought about that. My bad... :smiley: